deploying-an-existing-application-using-okteto-ui.md

```diff
@@ -0,0 +1,175 @@
+# Deploying An Existing Application Using Okteto UI
+
+In the previous blog post, you learned how to build a FastAPI CRUD application and deployed it to Okteto using the Okteto CLI tool. In this tutorial, you will be learning how to deploy the application directly from your Okteto dashboard.
+
+## Initial Setup
+
+Start by creating a fork of the application from [GitHub](https://github.com/okteto/fastapi-crud) and then clone it locally. In your local clone, set it up using the following commands:
+
+```bash
+$ cd fastapi-crud
+$ python3 -m venv venv && source venv/bin/activate
+(venv)$ pip install -r requirements.txt
+```
+
+Verify that the setup is complete by running:
+
+```bash
+python3 main.py
+```
+
+## Initial Deployment
+
+In the previous post, you defined an Okteto stack file to facilitate the deployment of your application from the CLI tool. In this section, you will be deploying the application from the dashboard.
```

Comment:
add a link to either the part of the post where we defined, in case they are not following through.

Reply:
sure.